src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -260,37 +255,33 @@ ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model) {
     } else {
         config.emplace("NPUW_FUNCALL_FOR_ALL", "YES");
     }
-    enable_npuw_dq_if_allowed(config, model);
     return config;
 }
 
 ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model,
                                       const std::optional<NPUDesc>& npudesc) {
-    // NB: Empty config for devices != NPU
-    if (!npudesc.has_value()) {
-        return { };
-    }
     auto config = get_default_common_config(model);
-    if (get_option<std::string>(config, "NPUW_FUNCALL_FOR_ALL").value_or("NO") == "YES") {
-        config.emplace("NPUW_PARALLEL_COMPILE", "YES");
-    }
-    if (npudesc.has_value() &&
-        npudesc->arch == "4000" &&
-        npudesc->max_tiles != -1) {
-        config.emplace("NPU_DPU_GROUPS", npudesc->max_tiles);
+    if (is_cw_compressed(model)) {
+        config.emplace("NPUW_DQ", "YES");
+    } else {
+        config.emplace("NPUW_PMM", "NO");
+        if (npudesc.has_value() &&
+            npudesc->arch == "4000" &&
+            npudesc->max_tiles != -1) {
+            config.emplace("NPU_DPU_GROUPS", npudesc->max_tiles);
+        }
```

Comment:
you set DPU groups here only if it is GQ.

In fact, this setting should be done regardless of the Q type.

Reply:
Done